config/monitor/config.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package monitor
+
+import (
+	"github.com/crossplane-contrib/terrajet/pkg/config"
+
+	"github.com/crossplane-contrib/provider-jet-azure/apis/rconfig"
+	"github.com/crossplane-contrib/provider-jet-azure/config/common"
+)
+
+const groupLog = "monitor"
+
+// Configure configures monitor group
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("azurerm_monitor_metric_alert", func(r *config.Resource) {
+		r.Kind = "MetricAlert"
+		r.ShortGroup = groupLog
+		r.References = config.References{
```

Comment:
What about configuring a referencer for the `scopes` [parameter](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/monitor_metric_alert)?

Reply:
The `StorageAccount` reference is used for the `scopes` parameter in the document. The `scopes` field is the reference to the resource to which the created metric will be applied, so this resource will not always be `StorageAccount`. As seen in the example manifest, this resource can also be a `PostgreSQLServer` resource. 

So this reference is actually a generic reference. In case of generic reference support to Terrajet, this issue can be discussed again. (I also added a TODO for this)